i18n/chs/src/vs/editor/contrib/find/browser/findWidget.i18n.json

```diff
@@ -6,10 +6,12 @@
 {
 	"label.closeButton": "关闭",
 	"label.find": "查找",
-	"label.matchesLocation": "第 {0} 个(共 {1} 个)",
+	"label.matchesLocation": "{0} / {1}",
 	"label.nextMatchButton": "下一个匹配",
 	"label.noResults": "无结果",
 	"label.previousMatchButton": "上一个匹配",
+	"label".allMatchButton": "选中所有匹配",
```

Comment:
@rebornix Do you mean that do not modify the localization file? Should I submit a PR to increase localization translation?